scene/gui/video_stream_player.cpp

```diff
@@ -281,6 +281,9 @@ void VideoStreamPlayer::play() {
 	set_process_internal(true);
 	last_audio_time = 0;
 
+	// We update the playback, to render the first frame immediately
```

Comment:
```suggestion
	// We update the playback, to render the first frame immediately.
```

Reply:
While at it, the comma can be removed:
```suggestion
	// We update the playback to render the first frame immediately.
```